docs/content/dagster-cloud/deployment/agents/customizing-configuration.mdx

```diff
@@ -97,9 +97,13 @@ compute_logs:
 
 ---
 
-## Streaming compute logs to AWS S3
+## Writing compute logs to AWS S3
 
-You can set up streaming compute logs to AWS S3 by configuring the [dagster_aws.s3.compute_log_manager](https://docs.dagster.io/\_apidocs/libraries/dagster-aws#dagster_aws.s3.S3ComputeLogManager) module. Note that by setting the `show_url_only` parameter to `true`, only the URL of the log file will be shown in the Dagster Cloud UI instead of fetching and displaying the full content.
+You can write compute logs to an AWS S3 bucket without requiring your S3 credentials by configuring the [dagster_aws.s3.compute_log_manager](https://docs.dagster.io/\_apidocs/libraries/dagster-aws#dagster_aws.s3.S3ComputeLogManager) module.
+
+You are also able to stream partial compute log files by configuring the log upload interval (in seconds) using the `upload_interval` parameter.
```

Comment:
I think there was some back-and-forth about using the word `streaming` here before. Should we also replace `stream` in this paragraph to be consistent?

Reply:
Ah no, that paragraph uses the same language as in the other sections on that page that talk about how to stream logs vs writing completed logs. I changed the title of the section to be "Writing" instead of "Streaming" for clarity.